app/assets/javascripts/remote_consoles/remote_console.js

```diff
@@ -0,0 +1,29 @@
+//= require jquery
```

Comment:
you don't need this, and move the file up by one folder, I won't expect any other JS code here